include/drivers/st25r3911b_nfc.rst

```diff
@@ -0,0 +1,84 @@
+.. _st25r3911b_nfc_readme:
+
+NFC Reader ST25R3911B
+#####################
+
+NFC Reader ST25R3911B can be used to read/write in ISO/IEC 14443 A compatible NFC tags,
+creating an NFC polling device. The library is intended to use with the NFC reader ST25R3911B
+Nucleo expansion board.
+
+NFC Reader ST25R3911B
+*********************
+
+The ST25R3911B is an Arduino shield which can be used to communicate over NFC.It can generate its own
```

Comment:
To be clear, ST25R3911B is the  NFC Reader chip, while X-NUCLEO-NFC05A1 is the name of the arduino-compatible shield.

Reply:
clarified (correctly, I hope ;) )